src/AppRoutes.tsx

```diff
@@ -43,6 +44,7 @@ const AppRoutes: FC = () => {
         <Route exact path="/search/:searchTerm" component={Search} />
         <Route exact path="/search/:searchTerm/:offset" component={Search} />
         <Route exact path="/logout" component={Logout} />
+        <Route exact path="/forbidden" component={Forbidden} />
```

Comment:
jeg syns det er litt rart å ha en egen side for forbidden, slik at hvem som helst kan gå hit bare for å få beskjed om at det er forbidden, uten at man har noen indikasjon på _hva_ som er forbidden.
syns det er mer naturlig at det er en blokk på det i `AppRoutes`, og at man fortsatt har den aktuelle URLen i browser når man får "Forbidden"

Reply:
Ja, jeg ser den. Men hvordan skal vi vite om vi har tilgang til en publikasjon når vi ikke gjør oppslag på publikasjonen før vi har rendret PublicationForm? Skal vi gjøre oppslag på publikasjonen i Approutes da?

Reply:
(vi må gjøre oppslag på publikasjonen for å finne owner)

Reply:
hmmm, det er sant...
da heller jeg mer mot å ha sjekken i `PublicationForm`, type
```
if (user.id !== publication.owner) {
    return <Forbidden />
}
```

Reply:
Ja, jeg gjorde det først, men da står det `Edit publication` som overskrift også står det forbidden under. Og syntes det så litt rart ut. Kan researche andre alternativ!

Reply:
flyttet overskriften, så da ble det ala det du tenkte 👍